pom.xml

```diff
@@ -20,7 +20,7 @@
     <parent>
         <groupId>io.confluent</groupId>
         <artifactId>common-parent</artifactId>
-        <version>7.2.0</version>
+        <version>7.2.6</version>
```

Comment:
this is already upgraded to `7.5.0` https://github.com/confluentinc/kafka-connect-datagen/blob/621d7f6106ed206f7774d45b92d8aa572615e56e/pom.xml#L23 . Kindly look into the merge conflicts.